pwiz_tools/Shared/zedgraph/ZedGraph/ZedGraphControl.cs

```diff
@@ -606,6 +606,8 @@ protected override void OnPaint( PaintEventArgs e )
 				base.OnPaint( e );
 
 				// Add a try/catch pair since the users of the control can't catch this one
+                //CONSIDER: Log the exception. Otherwise it can lead to some very hard to 
+                //  diagnose problems.
```

Comment:
You can certainly add a call to:
Trace.TraceWarning(@"Error painting: {0}", exception);
if you want